applications/cheap_flights/data_manager.py

```diff
@@ -34,26 +34,27 @@ def update_airline_name(airline_id) -> str:
         return airline_id
 
 
-def format_flight_route(flight_routes) -> str:
+def format_flight_route(flight_routes) -> list[str]:
     """ Format the flight route information into a string """
 
     def format_datetime(datetime_str):
-        date = datetime.fromtimestamp(datetime_str)
         """ Format datetime object as string 'dd/mm/yy at 00:00 hours'"""
+        date = datetime.fromtimestamp(datetime_str)
         return date.strftime('%d/%m/%y at %H:%M')
 
-    formatted_text = []
```

Comment:
Naming things is hard, but try to use more descriptive names when possible.

Reply:
noted and edited